Sources/Common/DataModel/DataSetAttributes/index.js

```diff
@@ -261,8 +264,8 @@ export function extend(publicAPI, model, initialValues = {}) {
     'activePedigreeIds',
   ]);
 
-  if (!model.arrays) {
-    model.arrays = {};
+  if (!initialValues.arrays) {
+    initialValues.arrays = {};
```

Comment:
should be done in `defaultValues()`

Reply:
`arrays` need to be in the model before the setters of active properties are called. thus, this part has been remplaced by pushing to the model `initialValues.arrays` before the call of the setters.